sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnSignaturesTest.java

```diff
@@ -78,6 +87,75 @@ public void process(ProcessContext c) {}
         sig.processElement().extraParameters().get(0), instanceOf(ProcessContextParameter.class));
   }
 
+  @Test
+  public void testBasicDoFnAllParameters() throws Exception {
```

Comment:
Can you just break the test so each method tests one thing? Even though you get _n_ methods, chances are someone is only going in to fix the one that broke so it is much better for the next person to come along.